documentation/smg/12_deploying_releases_to_GitHub.md

```diff
@@ -0,0 +1,89 @@
+# Deploying releases to GitHub
+
+## ci-release-deploy Repository
+
+A repository
+[ci-release-deploy](https://github.com/pace-neutrons/ci-release-deploy)
+has been created to hold the code that implements the deployment pipeline.
+This decision was made so that code is not duplicated between Herbert and Horace
+and to provide fast checkout times for the deploy pipelines.
+
+## Release Pipelines
+
+The release pipelines in ANVIL's Jenkins server should be used to create release
+candidates.
+These pipelines have a `Release-` prefix.
+The pipelines differ from the master builds in that they ensure the
+`-DHORACE_RELEASE_TYPE=release` is passed to CMake.
+It also keeps release candidates separate from the nightly master builds.
+
+## Deploy Pipelines
+
+The deploy pipelines created on ANVIL's Jenkins server should be used to deploy
+release candidates to GitHub.
+Horace and Herbert each have their own deploy pipeline name `Deploy`.
+When triggered, this pipeline takes the following parameters:
+
+- `tag_sha`: The SHA of the Git revision on which to create the release tag
+- `version_number`: The full version number of the release e.g. `3.4.1`
+- `release_job_ids`: The IDs of jobs that contain target release artifacts.
+This is a multiline string parameter whose input should have the form:
+
+  ```txt
+  <pipeline name> <build number>
+  <pipeline name> <build number>
+                ...
+  <pipeline name> <build number>
+  ```
+
+  where, for example,
+  `<pipeline name> <build number>` = `Release-Windows-10-2019b 12`.
+
+  This allows the deploy pipeline to copy release artifacts from those builds.
```

Comment:
"allows" sounds like we're giving permission. It gives it complete information necessary to copy the required artefacts from the set of all build pipelines (and supports addition of new release pipelines for different build targets)

Reply:
Would a delimited string be easier to type or paste into the parameter box? `<pipelinename>,<buildnumber>; <pipelinename>,<buildnumber>; ...`. You can ignore whitespace to handle the inclusion of spaces or newlines around the separators.